benchmark/testFiles/services/LogProcessor.ts

```
@@ -0,0 +1,26 @@
+    
+    async processLogs() {
+        const logs = await fs.promises.readFile(this.logFile, 'utf8');
+        const entries = logs.split('\n');
+        
+        let processedEntries = [];
+        for(let entry of entries) {
+            try {
+                const parsed = JSON.parse(entry);
+                processedEntries.push(parsed);
+            } catch {
+                continue;
```

Biome

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)